src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -141,18 +141,19 @@ private Runtime() {}
 
     /**
      * Initiates the <a href="#shutdown">shutdown sequence</a> of the Java Virtual Machine.
```

Comment:
```suggestion
     * Initiates the {@linkplain ##shutdown shutdown sequence} of the Java Virtual Machine.
```

Reply:
Good idea. I've updated these and a couple more relating to the shutdown sequence and termination. (There are lots more but in unrelated areas, so I left them for now.)